### ISSUE TITLE
Could not resolve host: virtualization.modern.ie

### ISSUE THREAD

**Issue by MatzFan** [xdissent/ievms#316]
Has the VM's host changed recently? `http://modern.ie` now resolves to `https://developer.microsoft.com/en-us/microsoft-edge/`?

Running the install command from the README results in:

```
$ curl -s https://raw.githubusercontent.com/xdissent/ievms/master/ievms.sh | bash
Checking for VirtualBox
Checking for Oracle VM VirtualBox Extension Pack
Pack no. 0:   Oracle VM VirtualBox Extension Pack
Building IE 6 VM
Checking for existing OVA at /Users/me/.ievms/IE6 - WinXP.ova
Downloading OVA ZIP from http://virtualization.modern.ie/vhd/IEKitV1_Final/VirtualBox/OSX/IE6_WinXP.zip to IE6_WinXP.zip (attempt 1 of 3)
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0curl: (6) Could not resolve host: virtualization.modern.ie
\nERROR: Failed to download http://virtualization.modern.ie/vhd/IEKitV1_Final/VirtualBox/OSX/IE6_WinXP.zip to /Users/me/.ievms/IE6_WinXP.zip using 'curl', error code (6)\n
```

The VM's (from IE8 up) are available on that site, for example IE8 on Win7 is at `https://az412801.vo.msecnd.net/vhd/VMBuild_20141027/VirtualBox/IE8/Windows/IE8.Win7.For.Windows.VirtualBox.zip` so I guess it may be possible to pick up the relevant URL somehow.

Sadly I was after IE6 - anyone know if it is still available anywhere?

**Comment by joe-bader** [comment]
+1

**Comment by Crease29** [comment]
Anyone has alternative URLs?

**Comment by MatzFan** [comment]
Answering my own question and for anyone similarly interested; IE6 & IE7 are still available thanks to good old web.archive.org: [IE6](https://web.archive.org/web/20140327213333/http://virtualization.modern.ie/vhd/IEKitV1_Final/VirtualBox/OSX/IE6_WinXP.zip) [IE7](https://web.archive.org/web/20140327213035/http://virtualization.modern.ie/vhd/IEKitV1_Final/VirtualBox/OSX/IE7_Vista.zip) checksum for IE6 OK, not verified IE7.

**Comment by xdissent** [comment]
Hey guys, I found the new URLs for the old vms and just released 0.3.2. Please reopen if that doesn't work for you.
